### ISSUE TITLE
Insufficient Color Contrast in Placeholder Text

### ISSUE THREAD

**Issue by nathandyer** [freedomofpress/securedrop-client#3378]
# Insufficient Color Contrast in Placeholder Text

**Component/Feature:** 

* Email list and email chat history.
* Login.

<img width="283" height="92" alt="Image" src="https://github.com/user-attachments/assets/41188e67-6dd3-46d3-896e-e344883fbc25" />

<img width="297" height="96" alt="Image" src="https://github.com/user-attachments/assets/39d07f8a-a105-498e-92cb-9f3b488d1bd5" />

<img width="369" height="230" alt="Image" src="https://github.com/user-attachments/assets/8ebeb17c-afb8-4d3a-9bd0-aabb0f410c88" />

## Summary
The placeholder text in input fields (“Search by name”, “Message”, “Username”, “Passphrase”, and “Two-Factor Code” ) is displayed using a light gray color on a white background, resulting in insufficient contrast.

This makes the placeholder text difficult to read for users with low vision or reduced contrast sensitivity. Since placeholder text often provides important contextual guidance, poor contrast can impact users’ ability to understand the expected input.

## Affected Elements
- Email List and email chat history: Search by name input and Message input.
Login: Username, Passphrase and Two-Factor Code

## WCAG Reference
- 1.4.3 - Contrast (Minimum)

## Test Environment
- Screen Reader: VoiceOver
- Browser: n/a
- Operating System: Mac

## Recommended Remediation
Ensure that placeholder text meets the minimum contrast requirements defined in WCAG 2.2:
Text (including placeholder text) must have a contrast ratio of at least 4.5:1 against the background (WCAG 1.4.3 – Contrast (Minimum)

Additionally (in the case of Search by name and Message), placeholder text should not be used as the sole means of providing instructions. Visible labels should always be provided.

## Implementation Guidance
Update the placeholder text color to meet a minimum contrast ratio of 4.5:1 against the background
Avoid using very light gray colors 
Use a darker neutral color that maintains readability while visually distinguishing placeholder text from user input

## Acceptance Criteria
- The issue is resolved for keyboard-only users.
- Screen readers correctly announce relevant content and controls.
- The interface meets the referenced WCAG success criterion.
- Manual accessibility testing confirms the issue is fixed.
